sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/implementation/models/HookInfo.java

```diff
@@ -132,4 +132,15 @@ public HookInfo setExternalLink(String externalLink) {
     public List<String> getAdmins() {
         return this.admins;
     }
+
+    /**
+     * Set the admins property: hook administrators.
+     *
+     * @param admins the admins value to set.
+     * @return the HookInfo object itself.
+     */
+    public HookInfo setAdmins(List<String> admins) {
```

Comment:
Should this be an unmodifiable list?

Reply:
This is input list, only in getter we apply `Collections.unmodifiableList(..)`